### PR TITLE
Initialized Project with PostgreSQL template

### LINKED ISSUE #1: Project Creation
# REQUIREMENTS
Project creation and initialization

# DESIGN

### PR DESCRIPTION
**Pull request by corps01**
Co-authored-by: dany-herz42 <danyherzdev@gmail.com>
Co-authored-by: DiegoSteve <diegosteve123321@gmail.com>
Co-authored-by: alanRazoMen <arazo@ucol.mx>

# Description
We initialized our Ror project using the PostreSQL template, which facilitates the implementation of the PostreSQL Database motor in the future. 

We used the following command:
**rails new checkClock -d postgresql**

### CODE REVIEW

File: checkClock/config/database.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# PostgreSQL. Versions 9.3 and up are supported.
```
Comment (YilKanda) [inline checkClock/config/database.yml:1]: This file shouldn't be uploaded to GitHub, instead you should put a file with the name database.example.yml, only with the suggested configuration.

**Review by YilKanda — CHANGES_REQUESTED** [review]

**Review by JuanequeX — DISMISSED** [review]
LGTM

just the same observation that Yil said, you can put the `database.yml` file in your gitignore and upload in this repository only the `database.example.yml`, because this is private data of your project that only you should see

File: checkClock/config/database.yml (hunk 1 above)
Comment (corps01) [reply checkClock/config/database.yml:1]: Ok added `database.yml` to gitignore and created `database.example,yml` file

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**